testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -780,7 +780,7 @@ class TestPBCFlag(object):
     @dec.skipif(parser_not_found('TRZ'),
                 'TRZ parser not available. Are you using python 3?')
     def setUp(self):
-        self.prec = 3
+        self.prec = 2
```

Comment:
I wrote these tests.. But a long time ago. It might be they're ill conditioned, ie calculating COM which is very close to 0, so errors seem large.